--- sys/synthesis/submodules/altera_asmi2_xip_controller.sv	2024-07-15 02:34:49.476724296 +0300
+++ sys/synthesis/submodules/altera_asmi2_xip_controller.sv	2024-07-15 02:50:24.750038794 +0300
@@ -611,7 +611,7 @@
 		.USE_PACKETS         (1),
 		.USE_FILL_LEVEL      (0),
 		.EMPTY_LATENCY       (3),
-		.USE_MEMORY_BLOCKS   (1),
+		.USE_MEMORY_BLOCKS   (0),
 		.USE_STORE_FORWARD   (0),
 		.USE_ALMOST_FULL_IF  (0),
 		.USE_ALMOST_EMPTY_IF (0)

--- sys/synthesis/submodules/sys_master_0.v	2024-07-15 02:34:47.396703537 +0300
+++ sys/synthesis/submodules/sys_master_0.v	2024-07-15 02:50:14.617938092 +0300
@@ -163,7 +163,7 @@
 		.USE_PACKETS         (0),
 		.USE_FILL_LEVEL      (0),
 		.EMPTY_LATENCY       (3),
-		.USE_MEMORY_BLOCKS   (1),
+		.USE_MEMORY_BLOCKS   (0),
 		.USE_STORE_FORWARD   (0),
 		.USE_ALMOST_FULL_IF  (0),
 		.USE_ALMOST_EMPTY_IF (0)

--- sys/synthesis/submodules/sys_onchip_memory2_0.v	2024-07-15 02:34:47.540704974 +0300
+++ sys/synthesis/submodules/sys_onchip_memory2_0.v	2024-07-15 02:49:59.685789671 +0300
@@ -71,7 +71,7 @@
   defparam the_altsyncram.byte_size = 8,
            the_altsyncram.init_file = INIT_FILE,
            the_altsyncram.lpm_type = "altsyncram",
-           the_altsyncram.maximum_depth = 10496,
+           the_altsyncram.maximum_depth = 2048,
            the_altsyncram.numwords_a = 10496,
            the_altsyncram.operation_mode = "SINGLE_PORT",
            the_altsyncram.outdata_reg_a = "UNREGISTERED",
